content/open-source/sigstore/cosign/how-to-sign-a-container-with-cosign.md

```diff
@@ -167,29 +121,33 @@ You should be able to now access your published container via your Docker Hub ac
 
 Now that the container is in a registry (in our example, it is in Docker Hub), you are ready to sign the container and push that signature to the registry.
 
-Make sure you are in the right directory for your Cosign key pair. 
+You will call your registry user name and container name with the following Cosign command. Note that we are signing the image in Docker Hub keylessly with Cosign.
```

Comment:
```suggestion
You will call your registry user name and container name with the following cosign command. Note that we are signing the image in Docker Hub keylessly with cosign.
```

Reply:
I think there are more lowercase cosigns than Cosigns in other guides in this PR? I'm not sure which we prefer though.

Reply:
Yeah, the command should be lower-case but the project should be title case